shakedown/cli/main.py

```diff
@@ -57,6 +68,42 @@ def cli(**args):
     if not args['no_banner']:
         echo(banner(), n=False)
 
+    if args['provisioner_url']:
+      echo('Provisioning DC/OS cluster...', d='step-maj')
+
+      echo('Checking provisioner URL...', d='step-min', n=False)
+      try:
+        requests.get(args['provisioner_url'])
```

Comment:
Would be useful to have a sanity check here -- maybe determine 'provisioner URL type'?)